package.json

```diff
@@ -46,12 +46,12 @@
     "react-icon-base": "^2.1.2"
   },
   "devDependencies": {
+    "@sambego/storybook-styles": "^1.0.0",
```

Comment:
please keep ordering

Reply:
It's alphabet ordering. It will change whenever someone changes dependencies and reinstalls project with `yarn` :(